src/operator/l2_normalization.cc

```diff
@@ -26,13 +26,22 @@
 namespace mxnet {
 namespace op {
 template<>
-Operator* CreateOp<cpu>(L2NormalizationParam param) {
-  return new L2NormalizationOp<cpu>(param);
+Operator* CreateOp<cpu>(L2NormalizationParam param, int dtype) {
```

Comment:
is it done this way elsewhere?

Reply:
https://github.com/apache/incubator-mxnet/pull/3011/files

Reply:
ok